legate/core/context.py

```diff
@@ -368,16 +367,14 @@ def wrapper(*args: Any, **kwargs: Any) -> Any:
 
         return wrapper
 
-    def create_task(
+    def create_manual_task(
         self,
         task_id: int,
         mapper_id: int = 0,
-        manual: bool = False,
         launch_domain: Optional[Rect] = None,
```

Comment:
I think we can make this a mandatory argument now. Presumably we'd have to move it before `mapper_id`.

Reply:
Very fortunately it is passed by kw everywhere it is currently used

Reply:
https://github.com/nv-legate/legate.core/pull/621/commits/da4546fb882a56fbfe32eec4fe9596abec44d54b